app/ide-desktop/lib/dashboard/src/modals/AboutModal.tsx

```diff
@@ -0,0 +1,138 @@
+/** @file Modal for confirming delete of any type of asset. */
+import * as React from 'react'
+
+import LogoIcon from 'enso-assets/enso_logo.svg'
+
+import * as textProvider from '#/providers/TextProvider'
+
+import * as aria from '#/components/aria'
+import Modal from '#/components/Modal'
+import ButtonRow from '#/components/styled/ButtonRow'
+import SvgMask from '#/components/SvgMask'
+import UnstyledButton from '#/components/UnstyledButton'
+
+// =================
+// === Constants ===
+// =================
+
+/** The duration of time for which the "copy" button indicates that the text has
+ * successfully been copied. */
+const CLEAR_COPIED_STATE_TIMEOUT_MS = 2_500
+
+// ==================
+// === AboutModal ===
+// ==================
+
+/** Props for a {@link AboutModal}. */
+export interface AboutModalProps {
+  readonly supportsLocalBackend: boolean
+}
+
+/** A modal for confirming the deletion of an asset. */
+export default function AboutModal(props: AboutModalProps) {
+  const { supportsLocalBackend } = props
+  const { getText } = textProvider.useText()
+  const [isCopied, setIsCopied] = React.useState(false)
+  const textContainerRef = React.useRef<HTMLTableSectionElement | null>(null)
+
+  const doCopy = () => {
+    const textContainer = textContainerRef.current
```

Comment:
why wouldn't you just build a string from the variables(window.versionInfo and so forth)?